### DISCUSSION TITLE
missing support in X11 for XUrgencyHint in response to bells

### DISCUSSION THREAD

**Discussion by nipsy** [ghostty-org/ghostty#4222] [Feature Requests, Ideas] [answered]
It seems that ghostty is missing the ability to raise an XUrgencyHint flag under X11 so my window manager can inform me of terminal activity on currently hidden from view terminals.  This one pretty much speaks for itself I think and I'm sure is simply an oversight.  :)

**Comment by nipsy** [comment]
I guess I should also mention that I don't have any ability to test this for macOS to see if it is handled on that end at all, however an urgency hint works there.  Wayland might also handle it, although, I haven't tested there either (mostly because Nvidia user and i3 works just fine still).

**Comment by rhodes-b** [comment, marked as answer]
See https://github.com/ghostty-org/ghostty/discussions/2710, I think this is a dupe

**Comment by nipsy** [comment]
Ha.  Searched for every version of urgency hint, but not bell.  Thanks.
